React/Views/RCTWrapperViewController.m

```diff
@@ -133,4 +140,18 @@ - (void)didMoveToParentViewController:(UIViewController *)parent
   }
 }
 
+- (UIImageView *)findHairlineImageViewUnder:(UIView *)view {
```

Comment:
Formatting is off.  Make sure to use 2 space indents and curly braces should be on their own line.